ios/Stockfish/Stockfish.h

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2017 Александр Лунев. All rights reserved.
 //
 
-#import "RCTBridgeModule.h"
+#import <React/RCTBridgeModule.h>
```

Comment:
With `#import "RCTBridgeModule.h"`, RCTBridgeModule.h will be imported twice in a project using react-native-stockfish, causing build error